ext/bestitamazonpay4oxid_oxorder.php

```diff
@@ -339,13 +339,13 @@ protected function _parentFinalizeOrder(oxBasket $oBasket, $oUser, $blRecalculat
      * Confirm Order details to Amazon if payment id is bestitamazon and amazonreferenceid exists
      * Update user details with the full details received from amazon
      *
-     * @param oxBasket $oBasket
-     * @param oxUser $oUser
-     * @param bool|false $blRecalculatingOrder
+     * @param \OxidEsales\Eshop\Application\Model\Basket $oBasket
+     * @param oxUser                                     $oUser
+     * @param bool|false                                 $blRecalculatingOrder
      *
      * @return int
      */
-    public function finalizeOrder(oxBasket $oBasket, $oUser, $blRecalculatingOrder = false)
+    public function finalizeOrder(\OxidEsales\Eshop\Application\Model\Basket $oBasket, $oUser, $blRecalculatingOrder = false)
```

Comment:
Could you use the `use` statement instead of the FQCN, to keep the code more readable.